packages/components/src/components/List/OrderedList.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "@xstyled/styled-components";
+import React, { ReactElement } from "react";
+import { Box } from "../../primitives/Box";
+import { ListItemProps } from "./ListItem";
+
+export interface OrderedListProps
+  extends Omit<React.HTMLAttributes<HTMLOListElement>, "color"> {
```

Comment:
I think we can remove this.

Reply:
Same here https://github.com/Localitos/pluto/pull/147#discussion_r1006881417